solr_conf/schema.xml

```diff
@@ -631,6 +631,7 @@ v         currently supported on types that are sorted internally as strings
    <field name="sourceResource_type_id" type="string" indexed="true" stored="true" multiValued="true" />
    <field name="sourceResource_type_type" type="string" indexed="false" stored="false" multiValued="true" />
    <field name="sourceResource_type_providedLabel" type="string" indexed="true" stored="true" multiValued="true" />
+   <field name="sourceResource_type_name" type="string" indexed="true" stored="true" multiValued="true" />
```

Comment:
The JSON-LD context maps `name` to `skos:prefLabel`. https://github.com/dpla/dpla_map/blob/map-4.0/lib/dpla/map/context.rb#L63